app/views/crn-enter.njk

```diff
@@ -13,7 +13,67 @@
 {% block content %}
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
+    <form method="POST" autocomplete="off" novalidate>
+    <input type="hidden" name="crumb" value="{{crumb}}"/>
     <h1 class="govuk-heading-l">Enter your customer reference number (CRN)</h1>
```

Comment:
We can consider using a 'govukFieldset' here. Something similar to

<img width="550" alt="image" src="https://user-images.githubusercontent.com/118533775/203568038-bd1b3235-8832-4c0e-a31b-b39378692545.png">